examples/quickstart/quickstart.ipynb

```diff
@@ -101,14 +101,14 @@
    "id": "966ce581",
    "metadata": {},
    "source": [
-    "## ☁️ Step 1: Connect to ZenML Cloud\n",
+    "## ☁️ Step 1: Connect to ZenML Pro\n",
```

Comment:
Everything in this file gets overwritten by the template action. This is already fixed though, just revert the changes